src/webapp-lib/policies/copyright.html

```diff
@@ -60,7 +60,7 @@
 
                     Deliver this Notice, with all items completed, to SageMath’s Designated Copyright Agent:<br><br>
 
-                    William Stein (Copyright Agent)<br>
+                    William Stein (Copyright (C) 2016, Sagemath Inc.
```

Comment:
This is the only one that would actually be visible to a user.  The parenthesis are wrong (no close).